src/workspacer.Bar/Widgets/BatteryWidget.cs

```diff
@@ -29,20 +30,20 @@ public override IBarWidgetPart[] GetParts()
             {
                 if (currentBatteryCharge <= LowChargeThreshold)
                 {
-                    return Parts(Part(currentBatteryCharge.ToString("#0%"), LowChargeColor));
+                    return Parts(Part(currentBatteryCharge.ToString("#0%"), LowChargeColor, null, null, FontName));
```

Comment:
These invocation might look cleaner if using [named arguments](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments).